drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/RuleModellerPageTest.java

```diff
@@ -117,7 +117,12 @@ public void testPrepareView() throws Exception {
         page.prepareView();
 
         verify(view).init(page);
+        verify(view).setupRuleModellerWidget(any(RuleModeller.class));
         verify(page).markAsViewed();
+
+        verify(translationService).format(GuidedDecisionTableErraiConstants.RuleModellerPage_InsertAnActionBRLFragment);
```

Comment:
Is there an equivalent test somewhere (or a new one?) that needs updating to show "Condition BRL Fragment" titles?